src/YesSql.Provider.SqlServer/YesSql.Provider.SqlServer.csproj

```diff
@@ -3,6 +3,13 @@
         <ProjectReference Include="..\YesSql.Core\YesSql.Core.csproj" />
     </ItemGroup>
     <ItemGroup>
-        <PackageReference Include="Microsoft.Data.SqlClient" Version="3.0.1" />
+        <PackageReference Include="Microsoft.Data.SqlClient" Version="4.1.0" />
```

Comment:
There were some breaking changes in this version so we decided not to update it yet

Reply:
rolled it back